explainaboard_client/__init__.py

```diff
@@ -1,5 +1,12 @@
-from explainaboard_api_client import models
+from __future__ import annotations
+
+import os
+from typing import Literal
+
 from explainaboard_client.client import ExplainaboardClient
-from explainaboard_client.config import Config
 
-__all__ = ["ExplainaboardClient", "Config", "models"]
+__all__ = ["ExplainaboardClient"]
+
+username: str | None = os.environ.get("EB_USERNAME")
+api_key: str | None = os.environ.get("EB_API_KEY")
```

Comment:
Don't use environment variables in the library. Especially, predefined environment variables is harmful because it has no strategy to avoid collisions among multiple libraries.